.github/workflows/kind.yml

```diff
@@ -81,8 +81,13 @@ jobs:
 
     - name: Verify Deployment Configuration
       run: |
+        make webhook
```

Comment:
Should this be after `make kind-deploy-controller-dev` since the webhook isn't running until then?

Reply:
I added webhook before propagator controller deployed because our default is --enable-webhooks=true , controller is looking for cert-manager when it initiates. If there isn't /tmp/k8s-webhook-server/serving-certs/tls.{crt,key}, image creating will be looping (fail)